llama-index-integrations/extractors/llama-index-extractors-marvin/llama_index/extractors/marvin/base.py

```diff
@@ -55,31 +45,18 @@ class MarvinMetadataExtractor(BaseExtractor):
     def __init__(
         self,
         marvin_model: Type[BaseModel],
-        llm_model_string: Optional[str] = None,
         **kwargs: Any,
     ) -> None:
         """Init params."""
-        import marvin
-        from marvin import ai_model
-
-        if not issubclass(marvin_model, ai_model):
-            raise ValueError("marvin_model must be a subclass of ai_model")
-
-        if llm_model_string:
-            marvin.settings.llm_model = llm_model_string
```

Comment:
is this no longer supported?

Reply:
The previous method no longer supports Azure OpenAI configuration, so I have removed this parameter from the code.

For more information, please refer to the documentation:  
https://www.askmarvin.ai/docs/configuration/settings/

**For those are using OpenAI:**

```python
import marvin

marvin.settings.openai.chat.completions.model = 'gpt-4o'
```

**For those using an Azure OpenAI model:**

```.env
MARVIN_PROVIDER=azure_openai
MARVIN_AZURE_OPENAI_API_KEY=<your-api-key>
MARVIN_AZURE_OPENAI_ENDPOINT="https://<your-endpoint>.openai.azure.com/"
MARVIN_AZURE_OPENAI_API_VERSION=2023-12-01-preview # or latest

MARVIN_CHAT_COMPLETIONS_MODEL=<your azure openai deployment name>
```

Reply:
got it. thanks for the confirmation!